sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -2070,12 +2069,30 @@ public void setViewHierarchyExporters(@NotNull final List<ViewHierarchyExporter>
     viewHierarchyExporters.addAll(exporters);
   }
 
-  public @NotNull IMainThreadChecker getMainThreadChecker() {
-    return mainThreadChecker;
+  /**
+   * @deprecated Use {@link SentryOptions#getThreadChecker} instead.
+   */
+  @Deprecated
+  @SuppressWarnings("InlineMeSuggester")
+  public @NotNull IThreadChecker getMainThreadChecker() {
```

Comment:
i added this method to reduce breaking changes, but since it's a major and the IThreadMainChecker (now IThreadChecker) is internal, we can remove it directly. 
Wdyt?

Reply:
yeah, I think it's safe to remove it